src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -206,7 +208,43 @@ QString AccessibleItem::accessibleDescription() const
         return QString();
     }
 
-    return readable(m_element->accessibleExtraInfo());
+    Selection selection = m_element->score()->selection();
```

Comment:
I would recommend moving the code below this into AccessibleRoot. In accessibleName() use accessibleRoot too

```
class AccessibleRoot
{
    bool isRangeSelection() const;
    QString rangeSelectionInfo();
};

AccessibleRoot* root = accessibleRoot();
if (root.isRangeSelection()) {
    return readable(root.rangeSelectionInfo());
}
    
return readable(m_element->accessibleExtraInfo());
```

Reply:
Looks good to me! @HemantAntony, please move the range selection code to `AccessibleRoot`.